fathom/src/driver.rs

```diff
@@ -277,7 +277,8 @@ impl<'surface, 'core> Driver<'surface, 'core> {
 
         let initial_buffer = binary::Buffer::from(buffer_data);
         let mut binary_context = binary::Context::new(initial_buffer);
-        let mut elab_context = elaboration::Context::new(&self.interner, &self.core_scope);
+        let mut elab_context =
+            elaboration::Context::new(format_file_id, &self.interner, &self.core_scope);
```

Comment:
Hmm, wouldn’t this be the `module_file_id`?

Reply:
It was a hack to ensure `item_env` was shared between the two `elab_context`s. `file_id` had to be initialized to *something* when `elab_context` was created, but it would immediately be overwritten by `elab_module`. I have thought of a more elegant way of handling it now though.